web_shop/src/main/java/com/createq/web_shop/facades/impl/DefaultProductCategoryFacade.java

```
@@ -25,11 +25,22 @@ public List<ProductCategoryDTO> getAll() {
         return productCategoryConverter.convertAll(productCategoryModelList);
     }
 
+    @Override
+    public ProductCategoryDTO getById(Long id) {
+    ProductCategoryModel productId=productCategorySerivce.getById(id);
+    if(productId==null) {
+        return null;
+    }
```

Please throw an exception here and handle it when you call this method.